src/connectors/Zengo/index.ts

```diff
@@ -0,0 +1,261 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
```

Comment:
what is this file based on?

Reply:
well based on... it's literally the wallet-connect implementation
https://github.com/blocknative/web3-onboard/blob/183ce59e08936849b026f11688429ffa77e93e2d/packages/walletconnect/src/index.ts#L1

this seemed the least time-intensive approach